src/mongroup.h

```diff
@@ -8,9 +8,12 @@
 #include "enums.h"
 #include "json.h"
 #include "string_id.h"
+#include "monster.h"
 
 // from overmap.h
 class overmap;
+// from monster.h
```

Comment:
Really? You feel the need to document that the `class monster` is defined in monster.h? And btw. you don't need the forward declaration at all, you're including monster.h anyway and thereby the declaration of monster as well!